desktop/ui/src/main/java/org/datacleaner/widgets/tree/SchemaTree.java

```diff
@@ -645,9 +645,9 @@ public String getToolTipText(MouseEvent event) {
     public Datastore getDatastore() {
         return _datastore;
     }
-    
+
     @Override
-    public void onDescriptorsUpdated(DescriptorProvider descriptorProvider) {
+    public synchronized void onDescriptorsUpdated(DescriptorProvider descriptorProvider) {
```

Comment:
Yes, I know. This problem will be fixed other way. Special branch is for this. I will remoce this change.